src/components/Comments/WriteComment.js

```diff
@@ -118,27 +127,39 @@ const WriteComment = ({
             />
           </>
         )}
-
         <TextInput
           testID="changeinput"
           ref={commentInputRef}
           placeholder={StringConstant.commentBoxDefaultPlaceholder}
+          style={[styles.text, styles.content(isViewOnly)]}
           placeholderTextColor={COLORS.blackgrey}
-          style={[styles.text, styles.content]}
           onChangeText={onChangeText}
           value={value}
           multiline
           textAlignVertical="center"
+          pointerEvents={isViewOnly ? 'none' : 'auto'}
         />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The TextInput component has received some updates. The style is now conditionally set based on the `isViewOnly` prop and the pointerEvents prop is also conditionally set. These changes seem to be well thought out and should improve the functionality of the component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->